Dockerfile

```diff
@@ -1,11 +1,18 @@
-FROM golang:1.8
+FROM golang:1.8 AS builder
 
-RUN mkdir -p /go/src/github.com/luizalabs/teresa
 WORKDIR /go/src/github.com/luizalabs/teresa
 COPY . /go/src/github.com/luizalabs/teresa
 
 RUN make build-server
 
+FROM debian:9-slim
+RUN apt-get update && \
+apt-get install ca-certificates -y &&\
+rm -rf /var/lib/apt/lists/*
+
+WORKDIR /go/src/github.com/luizalabs/teresa
```

Comment:
Maybe use `/app` as workdir

Reply:
I prefer to follow the go pattern

Reply:
IMO it's valid only in builder step

Reply:
Gotcha 👍

Reply:
Take a look if the cache dir is /var/lib/apt/lists or /var/cache/apt/archives (maybe both).